src/libsync/account.h

```diff
@@ -191,13 +191,19 @@ class OWNCLOUDSYNC_EXPORT Account : public QObject
 
     CredentialManager *credentialManager() const;
 
+    /**
```

Comment:
Not sure this function in combination with the private signal really prevents anyone from using it wrong again.

Reply:
Well, it certainly won't hurt either.